docs/documentation/latex.md

```diff
@@ -30,6 +30,24 @@ f
 ```
 \end{examplefigure}
 
+You can also mix math-mode and text-mode:
+
+\begin{examplefigure}{svg = true}
+```julia
+using CairoMakie
+CairoMakie.activate!() # hide
+
+f = Figure(fontsize = 18)
+
+ax = f[1,1] = Axis(f, 
```

Comment:
I would use the `Axis(f[1, 1])` syntax instead, I've been phasing out use of the other one over time